dev/packages/beats/aws-0.0.1/dataset/elb-logs/manifest.yml

```diff
@@ -3,67 +3,5 @@ release: ga
 type: logs
 streams:
 - input: logs
-  vars:
```

Comment:
Someone reviewing the AWS package will have to figure out which ones should be shared and which ones not.

Reply:
I diverged this PR from this one: https://github.com/elastic/package-registry/pull/337 (waiting for review). All unrelated files will disappear once the https://github.com/elastic/package-registry/pull/337 ) is fixed. I apologize, I didn't mention it in the description.

Reply:
@ruflin I've merged the mentioned PR. Number of affected files is 40 now (only fields files). Feel free to take a look.